### DISCUSSION TITLE
How to host it on github pages ?

### DISCUSSION THREAD

**Discussion by manewsletter** [JulianPrieber/littlelink-custom#362] [Questions]
Hi, do you have a solution to host littlelink-custom on a Github page ?

**Comment by JulianPrieber** [comment]
Since LittleLink Custom requires PHP to run, there is no way to run this on GitHub pages.
For this, you'll have to use [LittleLink](https://github.com/sethcottle/littlelink). This runs on GH pages but requires a bit of HTML knowledge sine you'll have to build it yourself.

Alternatively, you can just use any existing [public LittleLink Custom instance](https://littlelink-custom.llc.ovh/) online to create your personal links page.
